### DISCUSSION TITLE
"[WARN] [rootcoord/quota_center.go:238]" repetitive logs in milvus standalone.

### DISCUSSION THREAD

**Discussion by muazhari** [milvus-io/milvus#32756] [Q&A and General discussion] [answered]
What happens with my `milvusdb/milvus:2.4-latest-gpu` (c0c16db6524c) instance? And, do you know how to fix it?
![image](https://github.com/milvus-io/milvus/assets/39398937/db3fa310-2fe2-4ff5-ab8a-e0288bf01fb8)
All of my databases already have one ID field as the primary key each. That log occurs repetitively even when the container has just been run without any manual trigger to the Milvus.

Example one of my database:
![image](https://github.com/milvus-io/milvus/assets/39398937/14073c1c-239a-4a3d-86af-2af53ec5e723)

Full Logs:
https://gist.github.com/muazhari/3bcde6ac1363f4dea9f7f0b2b1a34d69/raw/9a70f2dc95cb845d2269bdb659aa5b6e3581ad39/gistfile1.txt

**Comment by yhmo** [comment, marked as answer]
This is a bug of v2.4.0. This warning is unnecessary. We removed it in the v2.4.1
Try the new version:
`docker pull milvusdb/milvus:v2.4.1-gpu`
